dev/build_release.py

```diff
@@ -633,9 +799,8 @@ def do_build(cls, options, build_number=None, container_builder=None):
   @classmethod
   def main(cls):
     parser = argparse.ArgumentParser()
-    cls.init_argument_parser(parser)
+    cls.init_argument_parser(parser, VALID_PLATFORMS)
```

Comment:
I'd leave this global rather than adding it as an argument.
You are already using it global in do_build anyway.